.gitignore

```diff
@@ -9,5 +9,6 @@
 *.csv
 !open_data_info.csv
 aws-s3/parcel-tmp
+aws-s3/scripts-ccao-data-raw-us-east-1/corner-lot-raw/
```

Comment:
This PR adjusts the location of the output data slightly, so we need to add the new subdirectory to the `.gitignore` to ensure that no data get committed.